public/js/showoff.js

```diff
@@ -1554,6 +1554,34 @@ function nextStep(updatepv)
   }
 }
 
+function prevSec(updatepv)
+{
+  $(currentSlide).find('video').each(function() {
+    console.log('Pausing videos on ' + currentSlide.attr('id'))
+    $(this).get(0).pause();
+  });
+
+  var curSec = currentSlide.attr('data-section');
+  var prevSec = $('li:has(a.navSection:contains('+curSec+'))')
```

Comment:
If we can verify that getAllSections returns the sections in order every time, it would be much cleaner to use that and iterate through the sections.  It would also be less coupled to the HTML and less likely to break if we make future changes.

Reply:
Yes, I agree.